### PR TITLE
Change keys to get host site

### PR DESCRIPTION
**Pull request by David-Sharpe**
The JobSearch Get Request was getting host site from the args with the key :host_site and the params being passed in are in camel case.  It should therefore look for :HostSite

### CODE REVIEW

File: lib/cb/requests/job_search/get.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@ def http_method
 
         def headers
           {
-            'HostSite' => args[:host_site] || Cb.configuration.host_site,
```
Comment (bowersaa) [inline lib/cb/requests/job_search/get.rb]: Was anybody using this? Is there any way to tell (I mean, it didn't do anything, right?)

Comment (David-Sharpe) [inline lib/cb/requests/job_search/get.rb]: I can't really say.  I added back this feature along with the new feature for now.

Comment (oochelz) [inline lib/cb/requests/job_search/get.rb]: Should there be a test?

File: lib/cb/requests/job_search/get.rb [hunk 2]
```diff
@@ -29,7 +29,7 @@ def http_method
...
+            'HostSite' => args[:host_site] || args[:HostSite] || Cb.configuration.host_site,
```
Comment (bowersaa) [inline lib/cb/requests/job_search/get.rb]: if the old one is supported, this is a minor update now?

File: lib/cb/requests/job_search/get.rb (hunk 1 above)
Comment (David-Sharpe) [inline lib/cb/requests/job_search/get.rb]: Tests added.

File: lib/cb/requests/job_search/get.rb (hunk 2 above)
Comment (David-Sharpe) [inline lib/cb/requests/job_search/get.rb]: Talked it over with Jeff.  Made it a patch update.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,7 @@ Version History
 ====
     * All Version bumps are required to update this file as well!!
 ----
+* 21.0.1 JobSearch calls gets the args[:HostSite] in addition to args[:host_site]
```
Comment (ausmith) [inline CHANGELOG.md]: Coordinate with @kamarsh1 as she is pushing 21.1.0.

File: spec/cb/requests/job_search/get_spec.rb [hunk 4]
```diff
@@ -23,7 +23,23 @@
...
+    context 'HostSite in args' do
+      let(:argument_hash) { { some_arg: 'some_arg', host_site: 'UK' } }
+      let(:headers) { {'HostSite' => 'UK', 'Accept' => 'application/json;version=3.0', 'fake_header' => 'fake'} }
+
+      it { expect(job_search.headers).to eq(headers)}
+    end
+    
+    context 'host_site in args' do
+      let(:argument_hash) { { some_arg: 'some_arg', host_site: 'IE' } }
+      let(:headers) { {'HostSite' => 'IE', 'Accept' => 'application/json;version=3.0', 'fake_header' => 'fake'} }
```
Comment (ausmith) [inline spec/cb/requests/job_search/get_spec.rb]: Where is `fake_header` coming from? I feel like I'm missing that bit of the code.

**Pull request merged**